src/htcondor_es/convert_to_json.py

```diff
@@ -588,23 +589,32 @@
     "FINISHED": "finished",
 }
 
-_launch_time = int(time.time())
+clogger = get_task_logger(__name__)  # Celery logger
 
 # Initialize aff_mgr
 aff_mgr = None
-try:
-    aff_mgr = AffiliationManager(
-        recreate=False,
-        dir_file=os.getenv(
-            "AFFILIATION_DIR_LOCATION",
-            AffiliationManager._AffiliationManager__DEFAULT_DIR_PATH,
-        ),
-    )
-except AffiliationManagerException as e:
-    # If its not possible to create the affiliation manager
-    # Log it
-    logging.error("There were an error creating the affiliation manager, %s", e)
-    # Continue execution without affiliation.
+__aff_err = False
+
+
+def __generate_aff_mgr():
+    global aff_mgr, __aff_err
+    if __aff_err:
+        return
+    try:
+        aff_mgr = AffiliationManager(
+            recreate=False,
+            dir_file=os.getenv(
+                "AFFILIATION_DIR_LOCATION",
+                AffiliationManager._AffiliationManager__DEFAULT_DIR_PATH,
+            ),
+        )
+    except AffiliationManagerException as e:
+        # If its not possible to create the affiliation manager
+        # Log it
+        __aff_err = True
+        logging.error("There were an error creating the affiliation manager, %s", str(e))
```

Comment:
here is a typical example where you can replace logger and clogger calls with single print statement.